posthog-node/src/feature-flags.ts

```diff
@@ -474,6 +479,15 @@ function matchProperty(
   }
 
   const overrideValue = propertyValues[key]
+  if (overrideValue == null && !NULL_VALUES_ALLOWED_OPERATORS.includes(operator)) {
+    // if the value is null, just fail the feature flag comparison
+    // this isn't an InconclusiveMatchError because the property value was provided.
+    if (warnFunction) {
+      warnFunction(`Property ${key}  cannot have a value of null/undefined with the ${operator} operator`)
```

Comment:
```suggestion
      warnFunction(`Property ${key} cannot have a value of null/undefined with the ${operator} operator`)
```
(unless the space was intentional...)